src/Labels/Labels.php

```diff
@@ -279,6 +286,22 @@ private function getGoodbitsLabels(): array
 		];
 	}
 
+	/**
+	 * Return labels - Active Campaign
+	 *
+	 * @return array<string, string>
+	 */
+	private function getActiveCampaignLabels(): array
+	{
+		return [
+			'activeCampaignInvalidEmailError' => \__('Enter a valid email address.', 'eightshift-forms'),
+			'activeCampaignDuplicateError' => \__('Email address already exists in the system.', 'eightshift-forms'),
+			'activeCampaign500Error' => \__('There was and internal server error. Please try again.', 'eightshift-forms'),
```

Comment:
```suggestion
			'activeCampaign500Error' => \__('There was an internal server error. Please try again.', 'eightshift-forms'),
```

Reply:
Implies error happened on Forms' side, not AC side 🤷 

Something like this?
```suggestion
			'activeCampaign500Error' => \__('There was an error with the service. Please try again.', 'eightshift-forms'),
```